src/packer.pkr.hcl

```diff
@@ -46,62 +46,68 @@ variable "skip_create_ami" {
   type        = bool
 }
 
-data "amazon-ami" "debian_bullseye" {
+data "amazon-ami" "windows" {
   filters = {
-    name                = "debian-11-amd64-*"
+    name                = "Windows_Server-2022-English-Full-Base-*"
     root-device-type    = "ebs"
     virtualization-type = "hvm"
   }
   most_recent = true
-  owners      = ["136693071363"]
+  owners      = ["amazon"]
   region      = var.build_region
 }
 
 locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
 
-source "amazon-ebs" "example" {
+source "amazon-ebs" "windows" {
   ami_block_device_mappings {
     delete_on_termination = true
     device_name           = "/dev/xvda"
     encrypted             = true
     volume_size           = 8
     volume_type           = "gp3"
   }
-  ami_name                    = "example-hvm-${local.timestamp}-x86_64-ebs"
+  ami_name                    = "windows-commando-hvm-${local.timestamp}-x86_64-ebs"
   ami_regions                 = var.ami_regions
   associate_public_ip_address = true
-  encrypt_boot                = true
-  instance_type               = "t3.small"
-  kms_key_id                  = var.build_region_kms
+  encrypt_boot                = false
+  instance_type               = "t2.large"
+  // kms_key_id                  = var.build_region_kms
```

Comment:
This line cannot be commented out, since we require our AMIs to be encrypted.  We will have to show you how to build AMIS via the GH Actions workflows so that you can utilize our encryption keys.

Reply:
@jsf9k i resolved some of the conversations. When would you be available to go through utilizing your encryption keys in GH actions for building AMIs

Reply:
The commented-out lines were reverted in commit b513f47bcaa2cba33fe96e338023a35922a4b8f9.